### DISCUSSION TITLE
Disable/Enable habits

### DISCUSSION THREAD

**Discussion by CodeCubeNeo** [iSoron/uhabits#1628] [Feature Requests]
I think it would be very useful to enable and disable certain habits. For example if you have a habit for something like skiing you would want to "hide" or "disable" that habit tracking during the year and enable it (with a simple switch i suppose) only during the winter.

**Comment by CodeCubeNeo** [comment]
Nevermind. I see that you can archive habits. Maybe just add the archive/unarchive option in the edit menu of the habit

**Comment by exanrauzer** [comment]
The option to archive habit do not solve the case.

There is no option to "suspend" habit for a period of time. Available option to mark days with hypen mark isn't well suited: 1. you need to mark every day with hypen, 2. days marked with hypen makes a positive streak - if you make spring/summer/autumn with hypen it will generate you record skiing streak.

From the other hand: if you chose to archive habit - you wil get 0% at the start of next winter, because archive do not suspend the habit (counts in background).
